test/models/club_test.exs

```diff
@@ -2,17 +2,42 @@ defmodule Gatherto.ClubTest do
   use Gatherto.ModelCase
 
   alias Gatherto.Club
+  alias Gatherto.Athlete
 
-  @valid_attrs %{description: "some content", title: "some content"}
+  @valid_attrs %{description: "some content",
+                 title: "some content",
+                 location: "Cleveland, OH"}
   @invalid_attrs %{}
 
-  test "changeset with valid attributes" do
-    changeset = Club.changeset(%Club{}, @valid_attrs)
-    assert changeset.valid?
-  end
+  describe "a changeset" do
+    test "with valid attributes" do
+      changeset = Club.changeset(%Club{}, @valid_attrs)
+      assert changeset.valid?
+    end
+
+    test "with invalid attributes" do
+      changeset = Club.changeset(%Club{}, @invalid_attrs)
+      refute changeset.valid?
+    end
+
+    test "without a unique title" do
+      Club.changeset(%Club{}, Map.merge(@valid_attrs, %{title: "same_title"})) |> Repo.insert!
+      changeset = Club.changeset(%Club{}, Map.merge(@valid_attrs, %{title: "same_title"}))
+        |> Repo.insert
+      assert {:error, _} = changeset
```

Comment:
I very much like this style of testing models. When there is no error, the output is really nice since it gives you a good idea of why it fails.